src/websocket/index.ts

```diff
@@ -21,7 +21,22 @@ interface Request {
   params: Params
 }
 
+// Add connection counter
+let activeConnections = 0
+
 export const onConnection = async (socket: WebSocket.WebSocket): Promise<void> => {
+  // Check max connections limit
+  if (activeConnections >= CONFIG.websocket.maxConnections) {
+    socket.close(1013, 'Maximum number of connections reached')
```

Comment:
Should we say `server busy, please try again after some time` ?

Reply:
try again later is closer to code 1003. Shall we change the code too>

Reply:
yes

Reply:
done